packages/deployer/utils/deployments.js

```diff
@@ -136,7 +136,11 @@ function getDeploymentFolder(info) {
 }
 
 function _populateDefaults(info = {}) {
-  return { ...DeploymentInfo, ...info };
+  return {
+    folder: info.folder || DeploymentInfo.folder,
+    network: info.network || DeploymentInfo.network,
+    instance: info.instance || DeploymentInfo.instance,
+  };
```

Comment:
This change is removing the possibility of renaming the default proxy contract name, if we want to keep it should be:
```suggestion
  return {
    folder: info.folder || DeploymentInfo.folder,
    network: info.network || DeploymentInfo.network,
    instance: info.instance || DeploymentInfo.instance,
    proxyContract: info.proxyContract || DeploymentInfo.proxyContract,
  };
```

Reply:
Ok, disregard this comment, as we had that configuration on the defaults here but I wasn't being used, so it works correctly. Going to push a fix to completely remove it from here and just leave as a default hardhat configuration.